.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+
+# Pull Request Template
+
+**script name** -
+
+**What have you Changed**
+
+what you changed in the codebase.write here
+
+
+### Issue no.(must) - #
+
+### Self Check(Tick After Making pull Request)
+
+- [ ] This issue was assigned to me.
+- [x] One Change in one Pull Request
```

Comment:
Let's keep the checkboxes unmarked by default. Let me PR contributors mark them when and if they have completed the tasks.